exporter/stackdriverexporter/stackdriverexporter_test.go

```diff
@@ -14,4 +14,120 @@
 
 package stackdriverexporter
 
-// TODO: Add tests.
+import (
+	"strings"
+	"testing"
+
+	"github.com/spf13/viper"
+)
+
+func TestStackdriverTraceExportersFromViper(t *testing.T) {
+	// we set location in our test configs to avoid trying to
+	// contact metadata service.
+	testCases := []struct {
+		name       string
+		config     map[string]interface{}
+		wantErr    string
+		traceLen   int
+		metricsLen int
+	}{
+		{
+			name: "empty config",
+		},
+		{
+			name: "no project",
+			config: map[string]interface{}{
```

Comment:
this test is not returning error...

Reply:
Travis runs on GCE, so the project is detected. I removed the test case.